packages/hint/docs/user-guide/concepts/connectors.md

```diff
@@ -1,71 +1,91 @@
 # Connectors
 
-A `connector` is the interface between the `hint`s and the website
-you are testing. It is responsible for loading the website and exposing
-all the information to `webhint` such as resources, network data, etc.
+A connector is the interface between a .hintrc's configured hints and
+the website you are testing. It is responsible for loading and rendering
+that site, then exposing collected information to webhint such as
+resources, network data, etc.
 
-To configure a connector you need to update your `.hintrc` file to
-make it look like the following:
+To use a connector, first install its package. The package name should
+start with `@hint/connector-`, `webhint-connector-`, or
+`@scope/webhint-connector-`. Then, add the package name to the
+`connector` object within your .hintrc file. Packages within the
+`@hint/` namespace (like, for example, `@hint/connector-local`) can be
+added using their short name.
 
 ```json
 {
     "connector": {
-        "name": "connectorName"
+        "name": "local"
     }
 }
 ```
 
-Where `connectorName` is the name of the connector.
+Otherwise, use the full package name. If you were to use the
+hypothetical connector `@orgName/webhint-connector-super-duper`, your
```

Comment:
Love the example 😊

Minor nit, if I recall correctly I think we were trying to keep scope names lowercase.

```suggestion
hypothetical connector `@orgname/webhint-connector-super-duper`, your
```